infrastructure/kube/templates/bitcoin/bcoin/.env.template

```diff
@@ -0,0 +1,2 @@
+http-api-key=
+bcoin-url=
```

Comment:
Is it used anywhere?

Reply:
We should remove the whole `bcoin` setup.

Reply:
Ack. I'll then hold on until this is addressed.

Reply:
7f082d1910f70fd87e71fe1ee26d14aeaeaa5273